### ISSUE TITLE
EOWG Re-Charter 2020 (in-progress)

### ISSUE THREAD

**Issue by slhenry** [w3c/strategy#211]
# Charter Review

**[New charter proposal](https://w3c.github.io/charter-drafts/eowg-2020.html)**
[EOWG Charter 2020 Additional Information](https://www.w3.org/WAI/EO/wiki/EOWG_Charter_2020_Additional_Information)
[timeline](https://www.w3.org/WAI/EO/wiki/EO_2020_Chartering#Timeline)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 
 - [x] Existing WG recharter

changes from previous charter:
* List of new deliverables in [Charter Changes](https://w3c.github.io/charter-drafts/eowg-2020.html#history)
* [Diff](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FWAI%2FEO%2Fcharter2017&doc2=https%3A%2F%2Fwww.w3.org%2FWAI%2FEO%2Fcharter2020)

Horizontal Reviews (intended to be checked only by horizontal reviewers when they are satisfied with the charter)
  - [x] Accessibility (a11y)
  - [x] Internationalization (i18n)
  - [x] Privacy
  - [x] Security

Where would charter proponents like to see issues raised?
[GitHub](https://github.com/w3c/charter-drafts/issues/new)

**Comment by brewerj** [comment]
- Draft Charter is available from Shawn here: [https://www.w3.org/WAI/EO/charter2020]
- Per Shawn, EOWG requests that this move forward for horizontal review requests. Shawn, you can add review-requested labels for that, and then track whether those are completed.
- ... and also requested advanced work notice be sent to AC. See pending question from me.

**Comment by himorin** [comment]
no comment from i18n.

**Comment by michael-n-cooper** [comment]
No comments from APA. Will discuss in WAI-CC. Over to @brewerj to complete accessibility horizontal review.

**Comment by slhenry** [comment]
@wseltzer & @samuelweiler - Anything I can do to help move along Privacy & Security review on this draft charter? Thanks!

**Comment by slhenry** [comment]
[https://github.com/w3c/strategy/issues/211 WAI-CC e-mail notice]

**Comment by cwilso** [comment]
This group should really be called "Accessibility Education and Outreach WG".  It is ambiguous if called EOWG.

**Comment by slhenry** [comment]
> This group should really be called "Accessibility Education and Outreach WG". It is ambiguous if called EOWG.

Hi @cwilso I would fully agree, _if_ not already established. Co-Chairs, Staff, and participants considered that carefully with the last re-charter, and considered options for the name and the acronym. We concluded that it would more likely cause more work and confusion overall. More info: [Group Name Q & A](https://www.w3.org/WAI/EO/wiki/EOWG_Charter_2020_Additional_Information#Group_Name_-_Accessibility_Education_and_Outreach_Working_Group_.28EOWG.29 )

**Comment by cwilso** [comment]
That still seems like it's very focused on not confusing the current participants; for anyone outside the WG, it's going to be confusing.

**Comment by slhenry** [comment]
@wseltzer & @samuelweiler @brewerj - These should be pretty easy. :-)

**Comment by samuelweiler** [comment]
No comments re: security nor privacy.

I greatly appreciate the explainer doc, though I'm skeptical of both of its Q&A conclusions.

**Comment by iherman** [comment]
Thanks to all!

**Comment by slhenry** [comment]
@cwilso  Thanks for additional input. I agree that it was focused more on people who already know the context of EOWG.

I’ve provided a broader [group name analysis]( https://www.w3.org/WAI/EO/wiki/EOWG_Charter_2020_Additional_Information#Issue:_Group_Name) of the issue, pros and cons, and options -- from multiple perspectives.

We’re seeking more input, and we welcome your input on other pros or cons, and on how strongly you feel about the importance of changing the group’s name.

**Comment by plehegar** [comment]
charter approved.
